custom_components/steam_wishlist/entities.py

```diff
@@ -68,7 +78,17 @@ def state(self) -> int:
 
     @property
     def extra_state_attributes(self):
-        return {"on_sale": self.on_sale}
+        # Added Upcoming Media Card compatibility
+        placeholders = {
+            'title_default': '$title',
+            'line1_default': '$rating',
+            'line2_default': '$price',
+            'line3_default': '$release',
+            'line4_default': '$genres',
+            'icon': 'mdi:arrow-down-bold',
+        }
+        data_list = [placeholders] + [game for game in self.games if game["sale_price"]]
```

Comment:
The filtering in the list comprehension isn't actually doing anything. Regardless if you've checked or not checked the `show_all_wishlist_items` this will return the entire wish list.  This is because you are converting `None` to a `str`, `"None"`, which is not falsey.

```python
>>> bool("None")
True
```